app/views/carts/show.html.erb

```diff
@@ -4,3 +4,10 @@
 Item: <%= Item.find(id).title %><br>
 Quantity: <%= quantity %><br>
 <% end %>
+
+<%- unless current_user == nil -%>
+  <%= button_to "Proceed to checkout"%>
```

Comment:
!!!!! dead button !!!!! needs to be updated after checkout/functionality is implemented